iceoryx2-bb/log/src/logger/console.rs

```diff
@@ -36,7 +36,9 @@ impl Default for Logger {
 }
 
 impl Logger {
-    pub const fn new() -> Self {
+    pub fn new() -> Self {
+        init_log_level();
```

Comment:
i think it's better to do lazy initialization in `get_log_level` and ensure all loggers get level by `get_log_level`. needn't call `init_log_level` everywhere.

Reply:
@xieyuschen Thanks, that's much better indeed.